controller/userController.js

```diff
@@ -0,0 +1,45 @@
+var user = require("../model/userModel");
+const bcrypt = require("bcrypt");
+
+module.exports = {
+  signUp: async (req, res) => {
+    if (!req.body.email) {
+      res.send({ responseCode: 404, responseMessage: "email is required !" });
+    }
+    var query = { email: req.body.email };
+    const result = await user.findOne(query);
+    try {
+      if (result) {
+        if (result.email == req.body.email) {
+          res.json({
```

Comment:
create common function which will called once from controller not multiple times